doc/fuzzing.md

```diff
@@ -146,23 +145,43 @@ Every single pull request submitted against the Bitcoin Core repo is automatical
 ## macOS hints for libFuzzer
 
 The default Clang/LLVM version supplied by Apple on macOS does not include
-fuzzing libraries, so macOS users will need to install a full version, for
-example using `brew install llvm`.
+fuzzing libraries, so macOS users will need to install a full version.
 
-You may also need to take care of giving the correct path for `clang` and
```

Comment:
Do I see correctly that the content hasn't chnaged here? It's annoying to review this reformatting when it doesn't have any effect.

Reply:
I've fixed a typo (`non-systems clang`) and rearranged the lines based on sentence structure

Reply:
Ok, fine to fix the typo but you should keep the structure. That was aiming to maintain the 80 character limit which is used throughout most of the text in this file and I think many people prefer using that limit.

Reply:
That limit likely caused us to miss the typo (splitting between related words), it's why I have also reordered it minimally to be structured by meaning instead.